packages/kbn-apm-synthtrace/src/scenarios/helpers/logs_scenario_opts_parser.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export interface LogsScenarioOpts {
+  isLogsDb: boolean;
+}
+
+export const parseLogsScenarioOpts = (
```

Comment:
Firstly i don't see the code in `run_synthtrace.ts` updated. Which the CLI won't accept this parameter at all.

Secondly, this logic should reside in `run_synthtrace.ts` as it will also provide help text in the CLI for our users to know what option do they have and what it does.

https://github.com/elastic/kibana/blob/508762e3b5c12311d539105184bfb2b9039bf3f3/packages/kbn-apm-synthtrace/src/cli/run_synthtrace.ts

Reply:
ah good catch, completely missed that its an existing option. I guess all good in that case